packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -52,6 +54,12 @@ class BuildInfo {
   /// Target platform for the build (e.g. android_arm versus android_arm64).
   final TargetPlatform targetPlatform;
 
```

Comment:
Add implications to the docs here and to the user visible argument help. 

For instance, versionName has no functional effect on the Play Store and on users' phones except for display. 

iTunes orders CFBundleShortVersionString first and then CFBundleVersion